edge-agent/docker/windows/arm32v7/base/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Base image for arm32v7 dotnet-runtime container
+FROM mcr.microsoft.com/windows/nanoserver/insider:10.0.17763.55
+ 
+ENV DOTNET_VERSION 2.1.5
+ENV DOTNET_DOWNLOAD_URL https://dotnetcli.blob.core.windows.net/dotnet/Runtime/$DOTNET_VERSION/dotnet-runtime-${DOTNET_VERSION}-win-arm.zip
```

Comment:
We won't officially support ARM32 on Windows before the .NET Core team does. So why not use their nightly images? E.g., `microsoft/dotnet-nightly:2.2-sdk-nanoserver-1809-arm32`. That way, we don't need to duplicate what they're already doing.

Reply:
These dockerfile instruction is provided by Jiri.  Let me check with him.

Reply:
Reply from Jiri:
You probably do not want the SDK base layer, you would want the runtime.
​Also, I have been told there was an issue in the 2.2 arm32 dotnet builds; perhaps it will not hit the hub, or perhaps it has been fixed, but tread carefully.

I will keep as-is for now since I believe Jiri did some tests before with this base image.

Reply:
My mistake referring to the SDK image...I should have said `runtime`. Anyway, I hate duplicating the .NET Core Dockerfiles, but if there really is a bug...  Not sure why our Dockerfile wouldn't have the same issue??

Reply:
You should now be able to use `mcr.microsoft.com/windows/nanoserver:1809` (and I'd recommend doing so wherever relevant) as these containers are now public. I'd also +1 what @damonbarry said here and at least try the dotnet nightly to see if the fix is present yet.

Reply:
is this docker image "mcr.microsoft.com/windows/nanoserver:1809" running with ARM32 architecture?  I can update to use 2.2.0-sdk-nanoserver-1809-arm32 from https://hub.docker.com/r/microsoft/dotnet-nightly/.  No matter which one I used, the problems are:
1. Even the image can be built, I don't know if all features can be run as expected.
2. Once the officical ARM dotnet image is available, we will use it instead.
As a result, I prefer not changing it for now.

What do you think, @myagley and @arsing?